scripts/add_content_to_db.py

```diff
@@ -18,7 +18,7 @@
     "See also the --language option.)",
     usage="""
     python add_content_to_db.py [-h] --csv CSV --domain DOMAIN [--language LANGUAGE]
-
+    
```

Comment:
```suggestion

```

Weird that traling whitespace got included? One of the recommit hooks should have caught this